CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
+* AKS patch versions `1.33.3`, `1.32.7`, and `1.30.11` are now available. These versions include bug fixes with no CVE patches.
+* [Istio-based service mesh add-on](https://learn.microsoft.com/azure/aks/istio-about) is now compatible with [AKS Long Term Support (LTS)](https://learn.microsoft.com/en-us/azure/aks/long-term-support) for Istio revisions asm-1-25+ and AKS versions 1.28+. Please note that not every Istio revision will be compatible with every AKS LTS version. It is recommended to review the [Istio add-on support policy](https://learn.microsoft.com/azure/aks/istio-support-policy)  for an overview of this feature's support. 
+* [Disabling SSH](https://learn.microsoft.com/azure/aks/manage-ssh-node-access?tabs=node-shell#disable-ssh-overview) on Windows node pools is now available.
+* Ubuntu 24.04 CVM is now enabled by default.
+* AzureLinuxV3 Kata VHD image is now available for customers to use with Kata containers, including 600Mi Pod Overhead for kata pods when customers don't set a pod limit.
+* [API Server Vnet Integration](https://learn.microsoft.com/azure/aks/api-server-vnet-integration) is now available in the following additional regions: centralus, austriaeast, chilecentral, denmarkeast, israelnorthwest, malaysiawest, southcentralus2, southeastus3, southeastus5, southwestus, and usgovtexas. For the latest list of supported regions, see the [API Server VNet Integration documentation](https://learn.microsoft.com/azure/aks/api-server-vnet-integration#limited-availability).
+* [OIDC (OpenID Connect)](https://learn.microsoft.com/en-us/azure/aks/use-oidc-issuer) is now enabled by default on new cluster creation for Kubernetes version 1.34 and above.
+* [Node Auto-provisioning](https://learn.microsoft.com/azure/aks/node-autoprovision?tabs=azure-cli) enabled clusters can use of maintenance windows for scheduling node image upgrades on the aksManagedNodeOSUpgradeSchedule channel. This change enables NAP node image upgrades to respect any defined aksManagedNodeOSUpgradeSchedule maintenance window.
+
+#### Bug Fixes
+
+* Fixed a bug where [ETag](https://azure.github.io/azure-sdk/general_design.html#conditional-requests) was not returned in [ManagedClusters](https://learn.microsoft.com/rest/api/aks/managed-clusters?view=rest-aks-2025-05-01) or [AgentPools](https://learn.microsoft.com/rest/api/aks/agent-pools?view=rest-aks-2025-05-01) responses in API versions 2024-09-01 or newer, even though the API specification said it would be.
+
+#### Behavioral Changes
+
+* All AKS Automatic clusters, and AKS Standard clusters that enabled [Deployment Safeguards](https://learn.microsoft.com/azure/aks/deployment-safeguards) via the safeguardsProfile, will now have a new DeploymentSafeguards resource created in the same resource group as the cluster. See [Use Deployment Safeguards](https://learn.microsoft.com/azure/aks/deployment-safeguards) for more information.
```

Comment:
is this an extension resource scoped under managedClusters? if yes, customer wouldn't see it outside the managedClusters in the resource group no? - @qpetraroia, @NickKeller

Reply:
changes are required - "All AKS Automatic clusters, and AKS Standard clusters that enabled [Deployment Safeguards](https://learn.microsoft.com/azure/aks/deployment-safeguards) via the safeguardsProfile, will now have a new `Microsoft.ContainerService/deploymentSafeguards` sub-resource created `managedClusters`. See [Use deployment safeguards](https://learn.microsoft.com/azure/aks/deployment-safeguards) for more information"